packages/app/src/cli/services/info.test.ts

```diff
@@ -238,9 +238,8 @@ describe('info', () => {
 
     // Then
     expect(result).toContain('Extensions with errors')
-    expect(result).toContain('📂 ui_extension    tmp/project/extensions/test-ui-extension')
```

Comment:
The fixing of the logic to let `output` relativize the path to the working directory leads to the output content being dependent on the temporary directory created for this test case. Since the `relativize` function is already tested with unit test, I refrained from stubbing `cwd`.